momentGW/base.py

```diff
@@ -137,6 +137,9 @@ class BaseGW(Base):
     thc_opts = dict(
         file_path=None,
     )
+    input_opts = dict(
+        file_path=None,
+    )
```

Comment:
Currently, functionally they are the same but I deliberately split it off from the `GW.thc_opts["file_path"]` just in case we want to have a method for importing ERIs into a calculation. Would just require an if statement depending on say the file name. Could be useful if you have a long calculation that may fail and don't want to do run the DFT calc. again? though this would require a few more things in the import.

Reply:
For CDERIs I would probably prefer this to be handled by the mean field on the PySCF side -- they have functionality to do that. I think it makes more sense for them to be loaded on the PySCF side than for us to add another keyword argument here. Thoughts?

I think that similarly, the file path for the THC integrals should go once there is a proper interface to their generation, but we can't afford that luxury yet obviously.

Reply:
I'm happy to take that direction with it. I just figured given I was putting in something that would import ERIs into the CDERI code, may as well make it general and have this conversation. But happy to use pyscf for all CDERI imports.

As you say, hopefully we can remove all import file paths with a proper THC integral integration, but will have to see.